app/controllers/reviews_controller.rb

```diff
@@ -26,12 +28,24 @@ def create
   def update
     @review = Review.find(params[:id])
     @review.update(review_params)
+    @airline = Airline.find(@review.airline_id)
     if @review.save
       flash[:notice] = 'Review successfully edited'
-      redirect_to airline_path
+      redirect_to airline_path(@airline)
     else
       flash[:notice] = 'Rating can\'t be blank'
-      render :new
+      redirect_to airline_review_path(airline_id: @airline.id, id: @review.id)
+    end
+  end
+  def destroy
+    @review = Review.find(params[:id])
+    @airline = Airline.find(@review.airline_id)
+    if @review.destroy
+      flash[:notice] = 'Review successfully deleted'
+      redirect_to airline_path(@airline)
+    else
+      flash[:alert] = "Something went wrong"
+      redirect_to airline_path(@airline)
```

Comment:
you're redirecting to the same place in both of these if/else statements, just move the redirect_to underneath the if statement so it's only being executed once